### PR TITLE
Feat: Update commit status on GitHub

### LINKED ISSUE #11: Create StatusUpdate class
We need to create a class that will use the GitHub API to update the commit statuses.

### PR DESCRIPTION
**Pull request by filippanilsson**
Fix #11: Implemented a GitStatusUpdate class that creates a JSONObject and sends it to the GitHub API to update the commit status 

Added tests

### CODE REVIEW

File: DD2480-CI/src/main/java/skeleton/ContinuousIntegrationServer.java [hunk 1]
```diff
@@ -38,7 +36,7 @@ public void handle(String target,
     // used to start the CI server in command line
     public static void main(String[] args) throws Exception
     {
-        Server server = new Server(8080);
+        Server server = new Server(80);
```
Comment (annamaartensson) [inline DD2480-CI/src/main/java/skeleton/ContinuousIntegrationServer.java:39]: Is there a reason for the change of port?

**Review by Frequinzy — APPROVED** [review]
LGTM

File: DD2480-CI/src/main/java/skeleton/ContinuousIntegrationServer.java (hunk 1 above)
Comment (Frequinzy) [reply DD2480-CI/src/main/java/skeleton/ContinuousIntegrationServer.java:39]: I agree, shouldn't it follow the convention from the Canvas assignment, aka. port 8000+<group number> so 8012 in our case

Comment (filippanilsson) [reply DD2480-CI/src/main/java/skeleton/ContinuousIntegrationServer.java:39]: Ah, that’s left over from when I tested the feature, and I forgot to change it back. 
(When I tried to start ngrok from port 8080 it said the port was ”already in use”.) I’ll fix it.

**Review by annamaartensson — APPROVED** [review]
LGTM

**Pull request merged**